### PR TITLE
Simple UDP logging setup + error_log to syslog

### PR DESCRIPTION
**Pull request by gilleyj**
Referencing 
https://imagineeasy.atlassian.net/browse/DEVOPS-14 
and
https://imagineeasy.atlassian.net/browse/DEVOPS-44

### CODE REVIEW

File: nginx-app/templates/default/easybib.com.conf.erb [hunk 1]
```diff
@@ -19,6 +19,9 @@ server {
     listen 80 <%=@listen_opts%>;
 
     access_log <%= @access_log %>;
+<% if !node.fetch('nginx-app', {}).fetch('logging',{}).fetch('address',{}).empty? -%>
```
Comment (till) [inline nginx-app/templates/default/easybib.com.conf.erb]: Can you assign it to the template rather than accessing `node` there?

**Comment by till** [comment]
@gilleyj I am gonna test and  make some changes for you.

**Comment by gilleyj** [comment]
@till, works now, web-dna.json or stack.json would initiate the change to access_log as such: 
['nginx-app']['access_log'] = 'syslog:server=127.0.0.1:23232'

 and the default for server error_log is as such:

['nginx-app']['error_log'] = 'syslog:server=unix:/dev/log error'

File: nginx-app/attributes/default.rb [hunk 2]
```diff
@@ -9,6 +9,14 @@
 default['nginx-app']['ppa'] = 'ppa:nginx/stable'
 default['nginx-app']['client_max_body_size'] = '5m'
 
+default['nginx-app']['error_log'] = 'syslog:server=unix:/dev/log error'
+# default['nginx-app']['error_log'] = '/var/log/nginx/error.log error'
+
+default['nginx-app']['access_log'] = 'syslog:server=127.0.0.1:23232'
+
+default['nginx-app']['logging']['port'] = '23232'
```
Comment (till) [inline nginx-app/attributes/default.rb]: ^^^ I think `logging` is now unused.

**Pull request merged**